src/arch/xtensa/configs/apollolake_defconfig

```diff
@@ -8,3 +8,7 @@ CONFIG_PERFORMANCE_COUNTERS=y
 CONFIG_COMP_SRC_SMALL=y
 CONFIG_COMP_TDFB=n
 CONFIG_COMP_TONE=n
+CONFIG_COMP_KPB=n
+CONFIG_COMP_SEL=n
+CONFIG_COMP_CROSSOVER=n
+CONFIG_COMP_DRC=n
```

Comment:
A change to be va_list leads to a build failure?
Something's not right here...

Reply:
@plbossart i've made the _log_message() macro heavier with this change and so it breaks the bss section for APL